ddp-workspace/projects/ddp-lms/src/index.html

```diff
@@ -20,7 +20,15 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
 
     <script src="assets/config/pepperConfig.js"></script>
-  </head>
+
+    <script async src="https://www.googletagmanager.com/gtag/js?id=DDP_ENV.projectGAToken"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
```

Comment:
Can we move this into app.module.ts?  I worry about loading it here globally.

Reply:
Coldn't get it work when I tried to move all statements. 
Moved gtag('js', new Date()); which seem to be fine.

Reply:
Was able to move the gtag initializing javascript from index.html to app.module.ts
Doesn't look clean but seem to work .